games-arcade/mrboom_libretro/mrboom_libretro-4.5_20181226.recipe

```diff
@@ -7,9 +7,9 @@ HOMEPAGE="http://mrboom.mumblecore.org/"
 COPYRIGHT="2001-2018 Remdy Software, the libretro team"
 LICENSE="MIT"
 REVISION="1"
```

Comment:
please also bump this to REVISION="2"

Reply:
Not sure I follow. This is not revision 2, it's revision 1 of version 4.5?